src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,127 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+    private final float loadFactor;
+    private int threshold;
+
+    private V nullKeyValue = null;
+    private boolean nullKeyPresent = false;
+
+    private static class Node<K, V> {
```

Comment:
The 'Node' class should not have an empty line at the beginning of its declaration. Please remove the empty line to comply with the coding standards.

Reply:
Fix the comment

Reply:
Not fixed